vmdb/app/models/ems_refresh/parsers/openstack_common/orchestration_stacks.rb

```
@@ -21,10 +22,14 @@ def stacks
         end
 
         def detailed_stacks
+          return [] unless @orchestration_service
           # TODO(lsmola) We need a support of GET /{tenant_id}/stacks/detail in FOG, it was implemented here
           # https://review.openstack.org/#/c/35034/, but never documented in API reference, so right now we
           # can't get list of detailed stacks in one API call.
           @orchestration_service.stacks.collect(&:details)
+        rescue Excon::Errors::Forbidden
+          # Orchestration service is detected but not open to the user
+          []
```

Should we be logging a message here or is it ok to silently ignore it?